src/platform/plugins/shared/data_view_editor/public/components/flyout_panels/flyout_panel.tsx

```diff
@@ -120,6 +120,9 @@ const componentStyles = {
   flyoutColumn: css({
     height: '100%',
     overflow: 'hidden',
+    ['@media only screen and (max-width: 767px)']: {
```

Comment:
Just a quick note that we should prefer EUI tokens vs hardcoded sizes for these styles. They offer some hooks that should be able to help us, more info [here](https://eui.elastic.co/docs/getting-started/theming/tokens/breakpoints/#useeuimaxbreakpointsize-and-useeuiminbreakpointsize-hook).

Reply:
💯 , also I think we should use `useEuiMaxBreakpoint(size)` here instead of `@media` 

https://eui.elastic.co/v106.2.0/docs/getting-started/theming/tokens/breakpoints/#useeuimaxbreakpointsize-and-useeuiminbreakpointsize-hook